src/series11/SwapArray.java

```diff
@@ -0,0 +1,29 @@
+package series11;
+
+import java.util.Scanner;
+
+public class SwapArray {
+ public static void main(String[] args)
+ {
+	 Scanner sc = new Scanner(System.in);
+	 System.out.println("Enter the array size");
+	 int n =sc.nextInt();
+	 int[] array = new int[n];
+	 System.out.println("Enter the array elements");
+	 for(int i = 0; i < n; i++)
+	 {
+		 array[i] = sc.nextInt();
+	 }
+	 for(int i = 0; i < n; i++)
```

Comment:
Also, you just had to swap the first and last elements of the array one time. Not n items. Inside the loop, you are swapping it again and again. You don't really have to do that. Keep the swapping part out of the loop as if you keep swapping it again and again, then if the n is even, then there would be no swap(2 swaps will result to original array) and when n is odd, only then it would world